<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slot Machine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Slotmachine_Label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Slot machine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RTDB_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Return to dashboard</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Slot1_Line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Slot2_Line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Slot3_Line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Roll_Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll (25$)</string>
    </property>
   </widget>
   <widget class="QLabel" name="Earn_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 177, 0);</string>
    </property>
    <property name="text">
     <string>Earn amount</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
